<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicalSpoon</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CFCEC6;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame_top_btns">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #576079;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame { 
  background-color: #CFCEC6;
  border: 5px solid #CFCEC6
} 
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame { 
  background-color: #CFCEC6;
  border: 5px solid #CFCEC6
} 
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6"/>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="table_widget_topic_list">
           <property name="styleSheet">
            <string notr="true">QTableWidget {	
	background-color: #ABC2CB;
	padding: 10px;
	border-radius: 5px;
	gridline-color: #DFE6ED;
	border-bottom: 1px solid #DFE6ED;
}
QTableWidget::item{
	border-color: #DFE6ED;
	padding-left: 5px;
	padding-right: 5px;
	gridline-color: #DFE6ED;
}
QTableWidget::item:selected{
	background-color: #C7CDDA;
}
QScrollBar:horizontal {
    border: none;
    background: #A3A8AD;
    height: 14px;
    margin: 0px 21px 0 21px;
	border-radius: 0px;
}
 QScrollBar:vertical {
	border: none;
    background: #A3A8AD;
    width: 14px;
    margin: 21px 0 21px 0;
	border-radius: 0px;
 }
QHeaderView::section{
	Background-color: #576079;
	max-width: 30px;
	border: 1px solid #576079;
	border-style: none;
    border-bottom: 1px solid #576079;
    border-right: 1px solid #576079;
}
QTableWidget::horizontalHeader {	
	background-color: #576079;
}
QHeaderView::section:horizontal
{
    border: 1px solid #DFE6ED;
	background-color: #576079;
	padding: 3px;
	border-top-left-radius: 7px;
    border-top-right-radius: 7px;
}
QHeaderView::section:vertical
{
    border: 1px solid #576079;
}
</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>TOPIC NUMBER</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>TOPIC DESCRIPTION</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame { 
  background-color: #CFCEC6;
  border: 5px solid #CFCEC6
} 
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="info_button_topic_list">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	border: 5px solid #ABC2CB;
	border-radius: 2px;	
	background-color: #ABC2CB;
	border-color:#576079;
}
QPushButton:hover {
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}
QPushButton:pressed {	
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}

QToolTip {	
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_selected_keywords">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Weighted topic list:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_topic_list">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {	
	background-color: #FFFFFF;
	border-radius: 5px;
	gridline-color: #FFFFFF;
	border-bottom: 1px solid #FFFFFF;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="info_button_selected_tag">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	border: 5px solid #ABC2CB;
	border-radius: 2px;	
	background-color: #ABC2CB;
	border-color:#576079;
}
QPushButton:hover {
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}
QPushButton:pressed {	
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}

QToolTip {	
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_selected_tag">
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Tag:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_edit_get_tag">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {	
	background-color: #FFFFFF;
	border-radius: 5px;
	gridline-color: #FFFFFF;
	border-bottom: 1px solid #FFFFFF;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="get_topic_list_push_button">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	border: 5px solid #ABC2CB;
	border-radius: 5px;	
	background-color: #ABC2CB;
}
QPushButton:hover {
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}
QPushButton:pressed {	
	background-color: #ABC2CB;
	border: 2px solid #ABC2CB;
}</string>
             </property>
             <property name="text">
              <string>Select weighted topic list</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame { 
  background-color: #CFCEC6;
  border: 5px solid #CFCEC6
} 
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #576079;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
